shell/scripts/extension/publish

```diff
@@ -266,6 +267,15 @@ for d in pkg/*/ ; do
 
     if [ -f "${HELM_INDEX}" ]; then
       UPDATE="--merge ${HELM_INDEX}"
+    elif [ "${GITHUB_BUILD}" == "true" ] && [[ -n ${GITHUB_SOURCE} ]]; then
+      GITHUB_SOURCE=$(echo -e "${GITHUB_SOURCE}" | sed 's/https:\/\///g')
+
+      # Check if git branch contains index.yaml
+      wget -P ${BASE_DIR} https://raw.githubusercontent.com/${GITHUB_SOURCE}/${GITHUB_BRANCH}/index.yaml
+
+      if [ -f "${HELM_INDEX}" ]; then
+        UPDATE="--merge ${HELM_INDEX}"
+      fi
```

Comment:
Is this necessary? It looks like the only thing that can happen is `UPDATE="--merge ${HELM_INDEX}"` which would get set in the 2 lines above.

Reply:
This is necessary for finding the `index.yaml`, which initially existed on the `main` branch but since moving to `gh-pages` will need to be fetched in order to generate the new chart information with the existing charts.